### PR TITLE
Improving guidelines on changelog 

### PR DESCRIPTION
**Pull request by JoshuaLai**
# What
Improving the guidelines on how to use the new `rush changelog` prompts

# Why
Including new guidelines to help improve clarity. 

# Process & policy checklist

- [x] I have updated the project documentation to reflect my changes if necessary.
- [x] I have read the [CONTRIBUTING](https://github.com/Azure/communication-ui-library/blob/main/CONTRIBUTING.md) documentation.

**Is this a breaking change?**

- [ ] This change causes current functionality to break.

### CODE REVIEW

File: docs/references/changelog-guidelines.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+
+To ease our release process, we need to follow some guidelines when adding a new changelog entry.
+`rush changelog` will give developers the ooportunity to create a change file to describe your changes. 
+The `comment` field is used to generate the changelog entry, so it's important to follow the guidelines below.
+
+## Type
+Select whether your change is a Major, Minor, or Patch (for more information on which to choose see [Semantic Versioning](https://semver.org/)).
+
+When to use each type:
+1. Prerelease: Changes that are not yet released to the public.
```
Comment (emlynmac) [inline docs/references/changelog-guidelines.md:11]: These types don't match the types in the paragraph above. Which is correct?

**Review by dmceachernmsft — APPROVED** [review]

File: docs/references/changelog-guidelines.md [hunk 2]
```diff
@@ -0,0 +1,40 @@
+# Changelog Guidelines
+
+To ease our release process, we need to follow some guidelines when adding a new changelog entry.
+`rush changelog` will give developers the ooportunity to create a change file to describe your changes.
```
Comment (vhuseinova-msft) [inline docs/references/changelog-guidelines.md:4]: 'oportunity' typo

File: docs/references/changelog-guidelines.md [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+## Describe changes
+Text field entry to describe the changes. The changelog entry to be in present tense, capitalized, and to NOT end with periods.
+
+
+This is what will be included in the changelog when a new version is released, so keep it concise and meaningful. See [Tips for writing meaningful changelog entries](../reviewer-notes/tips-for-writing-changelog-entries.md).
+
+### Examples
+```md
+- Adds functionality to open image gallery when user taps on an inline image
+- Starting new `workstream` or `feature`
```
Comment (vhuseinova-msft) [inline docs/references/changelog-guidelines.md:34]: Should we have more strict rules about Ving, V and V in third person singular form?

File: docs/references/changelog-guidelines.md (hunk 1 above)
Comment (JoshuaLai) [reply docs/references/changelog-guidelines.md:11]: Ah good catch the ones here are correct.

File: docs/references/changelog-guidelines.md (hunk 3 above)
Comment (JoshuaLai) [reply docs/references/changelog-guidelines.md:34]: We could, would you like to suggest / update the examples on the verbiage to follow

File: docs/references/changelog-guidelines.md [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
```
Comment (vhuseinova-msft) [inline docs/references/changelog-guidelines.md:34]:
```suggestion
- Start new `workstream` or `feature` 
```

File: docs/references/changelog-guidelines.md [hunk 5]
```diff
@@ -0,0 +1,40 @@
...
+
+## Describe changes
+Text field entry to describe the changes. The changelog entry to be in present tense, capitalized, and to NOT end with periods.
+
+
+This is what will be included in the changelog when a new version is released, so keep it concise and meaningful. See [Tips for writing meaningful changelog entries](../reviewer-notes/tips-for-writing-changelog-entries.md).
+
+### Examples
+```md
+- Adds functionality to open image gallery when user taps on an inline image
```
Comment (vhuseinova-msft) [inline docs/references/changelog-guidelines.md:33]:
```suggestion
- Add functionality to open image gallery when user taps on an inline image
```

File: docs/references/changelog-guidelines.md [hunk 6]
```diff
@@ -0,0 +1,40 @@
...
+- Starting new `workstream` or `feature` 
+- Adding to `workstream` or `feature`
```
Comment (vhuseinova-msft) [inline docs/references/changelog-guidelines.md:35]:
```suggestion
- Add `functionality` to `workstream` or `feature` 
```

File: docs/references/changelog-guidelines.md [hunk 7]
```diff
@@ -0,0 +1,40 @@
...
+
+
+This is what will be included in the changelog when a new version is released, so keep it concise and meaningful. See [Tips for writing meaningful changelog entries](../reviewer-notes/tips-for-writing-changelog-entries.md).
+
+### Examples
+```md
+- Adds functionality to open image gallery when user taps on an inline image
+- Starting new `workstream` or `feature` 
+- Adding to `workstream` or `feature` 
+- Fixes app crash when user taps on call button
```
Comment (vhuseinova-msft) [inline docs/references/changelog-guidelines.md:36]:
```suggestion
- Fix for app crash when user taps on call button
```

File: docs/references/changelog-guidelines.md [hunk 8]
```diff
@@ -0,0 +1,40 @@
...
+- Improves performance of the app by reducing the number of API calls
```
Comment (vhuseinova-msft) [inline docs/references/changelog-guidelines.md:37]:
```suggestion
- Improve performance of the app by reducing the number of API calls
```

**Review by vhuseinova-msft — APPROVED** [review]

**Pull request merged**